src/litestorepkg/lib/api_v8.nim

```diff
@@ -0,0 +1,1407 @@
+import
+  asynchttpserver,
+  strutils,
+  sequtils,
+  cgi,
+  strtabs,
+  pegs,
+  json,
+  os,
+  uri,
+  tables,
+  times,
+  re,
```

Comment:
Err... can't allow Nim's re module, as it loads libpcre dynamically by default, and LiteStore must remain a self-contained, statically-compiled single executable. Use [pegs ](https://nim-lang.org/docs/pegs.html)instead of regular expressions!

(yes, you could statically link PCRE like I did in other projects but I'd rather avoid that if possible, unless you really need regex... but I think you can use pegs for matching wiki links etc.)